tools/dashboard/package.json

```diff
@@ -10,30 +10,30 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.1.2",
-    "@angular/cdk": "^6.4.3",
-    "@angular/common": "^6.1.2",
-    "@angular/compiler": "^6.1.2",
-    "@angular/core": "^6.1.2",
-    "@angular/forms": "^5.0",
-    "@angular/material": "^6.4.3",
-    "@angular/platform-browser": "^6.1.2",
-    "@angular/platform-browser-dynamic": "^6.1.2",
-    "@angular/router": "^6.1.2",
+    "@angular/animations": "^7.0.0-beta.4",
+    "@angular/cdk": "^7.0.0-beta.0",
+    "@angular/common": "^7.0.0-beta.4",
+    "@angular/compiler": "^7.0.0-beta.4",
+    "@angular/core": "^7.0.0-beta.4",
+    "@angular/forms": "^7.0.0-beta.4",
+    "@angular/material": "^7.0.0-beta.0",
+    "@angular/platform-browser": "^7.0.0-beta.4",
+    "@angular/platform-browser-dynamic": "^7.0.0-beta.4",
+    "@angular/router": "^7.0.0-beta.4",
     "@swimlane/ngx-charts": "^9.0.0",
-    "angularfire2": "^5.0.0-rc.11",
+    "angularfire2": "^5.0.0-rc.12",
     "core-js": "^2.5.7",
-    "d3": "^5.5.0",
-    "firebase": "^5.3.1",
-    "rxjs": "^5.6.0-forward-compat.5",
+    "d3": "^5.7.0",
+    "firebase": "^5.4.2",
+    "rxjs": "^6.3.1",
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.6.8",
-    "@angular/cli": "^6.1.3",
-    "@angular/compiler-cli": "^6.1.2",
-    "@angular/language-service": "^6.1.2",
-    "@types/node": "^10.5.8",
-    "typescript": ">=2.7.0 <2.8.0"
+    "@angular-devkit/build-angular": "^0.7.5",
+    "@angular/cli": "^6.1.5",
+    "@angular/compiler-cli": "^6.1.6",
+    "@angular/language-service": "^6.1.6",
+    "@types/node": "^10.9.4",
+    "typescript": "^2.9.x"
```

Comment:
Not upgrading to `^3.0.0` because otherwise there will be more peer dependency warnings from `tsickle` and `@ngtools/webpack`.